### PR TITLE
Thunder sisrightsizing

### LINKED ISSUE #806: Add a new API to verify Oracle Cloud software infracstructure service rightsizing
Add a new API
/oracle-cloud/sis-rightsizing/{ids}

to identify the list of Oracle Cloud oversized databses

### PR DESCRIPTION
**Pull request by anbrenna**
Fix #806

### CODE REVIEW

File: thunder-service/database/ercole_databases_test.go [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+				},
+			},
+			Archived: false,
+		},
+		{
+			Hostname: "test-small",
+			Info: model.Info{
+				CpuThreads: 2,
+			},
+			/*
```
Comment (cirelli94) [inline thunder-service/database/ercole_databases_test.go:146]: Would you mind to remove comments?

File: fixture/test_thunderservice_mongohostdata_08.json [hunk 2]
```diff
@@ -0,0 +1,131 @@
+{
```
Comment (cirelli94) [inline fixture/test_thunderservice_mongohostdata_08.json:1]: Is this file needed?

**Review by cirelli94 — CHANGES_REQUESTED** [review]

**Comment by cirelli94** [comment]
.. And would you mind to do a rebase from master?

**Review by cirelli94 — APPROVED** [review]

File: fixture/test_thunderservice_mongohostdata_08.json (hunk 2 above)
Comment (cirelli94) [reply fixture/test_thunderservice_mongohostdata_08.json:1]: Would you mind to delete this file? @anbrenna it's not needed

Comment (anbrenna) [reply fixture/test_thunderservice_mongohostdata_08.json]: done
